CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGE LOG
 
+## v3.0.0 ([#77](https://github.com/omise/omise-dotnet/pull/77))
```

Comment:
@AnasNaouchi this is not a major release. `2.10.0` would be more appropriate.

Reply:
Yeah I actually used math logic instead of semantic versioning in my head , 2.9.0+0.1=3.0.0 🤦